content/scout/policy/index.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Get started with policy in Docker Scout
+keywords: scout, supply chain, vulnerabilities, packages, cves, policy
+description: |
+  Policies in Docker Scout lets you define supply chain rules and thresholds
+  for your artifacts, and track how your artifacts perform against those
+  requirements over time
+---
+
+> **Early Access**
+>
+> Docker Scout is in [early access](/release-lifecycle/#early-access-ea). The
```

Comment:
**question:** Is there a way to consume feature flags in docs and change the content? We just [merged a PR](https://github.com/docker/pinata/pull/23521) that adds the ability to hide all early access messaging for scout in DD. It's controlled by the `ScoutGeneralAvailability` flag but I don't see it consumed here so we will need to keep place manually updated (and any other place in docs that has the same messaging).

Reply:
We don't have this added at the moment, but we could add some kind of feature-state toggle in the site config, and make these render conditionally. Let me add an item to look into it, but no big deal if I have do this manually for now.